### DISCUSSION TITLE
QVirtualScroll for CSS grid items

### DISCUSSION THREAD

**Discussion by PaulCombal** [quasarframework/quasar#11962] [General - Components / Directives / etc] [answered]
Hello, I have code as such:

```
  <div class="row">
    <q-virtual-scroll
        :items-size="items_count"
        :items-fn="getItems"
    >
      <template v-slot="{ item, index }">
        <div class="col-3">
          <media-selector-card :key="item.id" :model-value="item" allow-delete></media-selector-card>
        </div>
      </template>
    </q-virtual-scroll>
  </div>
```
My goal here is to have a virtual scroll area for many QCard items, arranged with CSS, preferrably 3 or 4 cards on the same row. The generated DOM about what I expected:
![image](https://user-images.githubusercontent.com/5382590/148692297-75bde51c-e7d3-49ec-8214-e87947b29e75.png)

But the item width is 0: the div with `.col-3` has a width of 0. I suspect this is because the `.col` tag is not a direct children of the `.row`. I tested without virtual scrolling and the results are great.

Is there a way around this, am I using this wrong?

Thank you

**Comment by hawkeye64** [comment, marked as answer]
Personally, I would use QIntersection unless there are specific reasons you prefer the QVirtualScroll.

**Reply by PaulCombal** [reply]
That's what I need, thank you very much for your answer. 

I feel like these two components are somehow connected. Would it be out of place if there were a link between them in the documentation?

Thank you again

**Reply by hawkeye64** [reply]
Not really connected, but achieve similar goals.
